lib/ansible/modules/windows/win_get_url.ps1

```diff
@@ -19,120 +20,186 @@
 # WANT_JSON
 # POWERSHELL_COMMON
 
+
+Function CheckModified-File($url, $dest, $headers, $credentials, $timeout, $use_proxy, $proxy) {
```

Comment:
So if `use_proxy` is set to false, your `$proxy` value should be `$null`. That way you don't have to check it in the function just assign it to the `Proxy` property